api/v1alpha1/event.go

```diff
@@ -1,22 +1,23 @@
 package v1alpha1
 
 const (
-	AnnotationKeyEventActor                  = "event.kargo.akuity.io/actor"
-	AnnotationKeyEventProject                = "event.kargo.akuity.io/project"
-	AnnotationKeyEventPromotionName          = "event.kargo.akuity.io/promotion-name"
-	AnnotationKeyEventPromotionCreateTime    = "event.kargo.akuity.io/promotion-create-time"
-	AnnotationKeyEventFreightAlias           = "event.kargo.akuity.io/freight-alias"
-	AnnotationKeyEventFreightName            = "event.kargo.akuity.io/freight-name"
-	AnnotationKeyEventFreightCreateTime      = "event.kargo.akuity.io/freight-create-time"
-	AnnotationKeyEventFreightCommits         = "event.kargo.akuity.io/freight-commits"
-	AnnotationKeyEventFreightImages          = "event.kargo.akuity.io/freight-images"
-	AnnotationKeyEventFreightCharts          = "event.kargo.akuity.io/freight-charts"
-	AnnotationKeyEventStageName              = "event.kargo.akuity.io/stage-name"
-	AnnotationKeyEventAnalysisRunName        = "event.kargo.akuity.io/analysis-run-name"
-	AnnotationKeyEventVerificationPending    = "event.kargo.akuity.io/verification-pending"
-	AnnotationKeyEventVerificationStartTime  = "event.kargo.akuity.io/verification-start-time"
-	AnnotationKeyEventVerificationFinishTime = "event.kargo.akuity.io/verification-finish-time"
-	AnnotationKeyEventApplications           = "event.kargo.akuity.io/applications"
+	AnnotationKeyEventPrefix                 = "event.kargo.akuity.io/"
+	AnnotationKeyEventActor                  = AnnotationKeyEventPrefix + "actor"
+	AnnotationKeyEventProject                = AnnotationKeyEventPrefix + "project"
+	AnnotationKeyEventPromotionName          = AnnotationKeyEventPrefix + "promotion-name"
+	AnnotationKeyEventPromotionCreateTime    = AnnotationKeyEventPrefix + "promotion-create-time"
+	AnnotationKeyEventFreightAlias           = AnnotationKeyEventPrefix + "freight-alias"
+	AnnotationKeyEventFreightName            = AnnotationKeyEventPrefix + "freight-name"
+	AnnotationKeyEventFreightCreateTime      = AnnotationKeyEventPrefix + "freight-create-time"
+	AnnotationKeyEventFreightCommits         = AnnotationKeyEventPrefix + "freight-commits"
+	AnnotationKeyEventFreightImages          = AnnotationKeyEventPrefix + "freight-images"
+	AnnotationKeyEventFreightCharts          = AnnotationKeyEventPrefix + "freight-charts"
+	AnnotationKeyEventStageName              = AnnotationKeyEventPrefix + "stage-name"
+	AnnotationKeyEventAnalysisRunName        = AnnotationKeyEventPrefix + "analysis-run-name"
+	AnnotationKeyEventVerificationPending    = AnnotationKeyEventPrefix + "verification-pending"
+	AnnotationKeyEventVerificationStartTime  = AnnotationKeyEventPrefix + "verification-start-time"
+	AnnotationKeyEventVerificationFinishTime = AnnotationKeyEventPrefix + "verification-finish-time"
+	AnnotationKeyEventApplications           = AnnotationKeyEventPrefix + "applications"
 )
 
 const (
```

Comment:
I would like to rename these to `EventType*` rather than `EventReason*` and have them use a new concrete type alias `type EventType string` to make things more clear. This is entirely optional and I didn't do it because I wasn't sure if we were ok breaking the API of this module

Reply:
DM'ed you.